src/frontend/src/lib/components/ui/Alert.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+	import IconInfo from '$lib/components/icons/lucide/IconInfo.svelte';
+
+	export let alertType: 'info' | 'light-warning' = 'info';
```

Comment:
light-warning? not just warning?

Reply:
`light-warning`, because we have a `warning` level according to design, that is different

Reply:
Yeah but `alertType` is not about the style but about the intent no?

Reply:
maybe I shall rename the prop with `componentStyle`, where `component` is what better name we can find instead of `Alert` eheh

Any suggestion?